common/src/main/java/com/wynntils/features/combat/QuickCastFeature.java

```diff
@@ -159,11 +159,18 @@ public void onTick(TickEvent e) {
         if (!Models.WorldState.onWorld()) return;
 
         // Clear spell after the 40 tick timeout period
-        if (spellCountdown > 0 && --spellCountdown <= 0) {
-            spellInProgress = SpellDirection.NO_SPELL;
+        if (spellCountdown > 0) {
+            spellCountdown--;
+            if (spellCountdown == 0) {
```

Comment:
We can probably still keep the <= just in case

Reply:
There is no other way to decrease spellCountdown? But sure, if you want to be extra defensive..